thirdparty/github.go

```diff
@@ -298,7 +314,7 @@ func getInstallationToken(ctx context.Context, owner, repo string, opts *github.
 		return "", errors.Wrap(err, "creating token")
 	}
 	if token == "" {
-		return "", errors.New("token is empty")
+		return "", missingTokenError
```

Comment:
Do we expect this to be temporary re Bynn's project, in which case do we have a follow up ticket we can link here or comment on to remove all of the DebugWhen statements?

Reply:
I asked for this to be silenced temporarily because it'll be loud in prod until we have the app installed
Jonathan could you link EVG-19966? And I can erase them once it's ready

Reply:
done